src/lib/common/ConfirmationModal.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+    import Modal from "./Modal.svelte";
+    import { createEventDispatcher } from "svelte";
+    const dispatch = createEventDispatcher();
+
+    export let open: boolean = false;
+    export let displayEscapeButton: boolean = false;
+    export let canCloseByClickingBackground: boolean = false;
+    export let title: string = "Confirm";
+    export let message: string = "Message";
```

Comment:
```suggestion
    export let title: string;
    export let message;
```

this'll force the caller to set both props